### PR TITLE
Fixing the Windows save problem

### LINKED ISSUE #36: Saving KEGG graph on Windows fails
**Describe the bug**
saving a KEGG grahp falis with ImproperImageHeader Error. 

** R Session Information:**
R version 3.6.3 (2020-02-29)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows 10 x64 (build 17134)

### PR DESCRIPTION
**Pull request by rix133**
On windows  the image file from KEGG servers is not properly downloaded with the default method="w"
fixes #36

### CODE REVIEW

**Comment by rix133** [comment]
The second commit d19b206 might be a more general problem and handling it need more thought.

File: R/visualization_functions.R [hunk 1]
```diff
@@ -659,6 +661,7 @@ download_kegg_png <- function(pw_url, f_path, quiet = TRUE) {
   res <- tryCatch({
     cond <- utils::download.file(url = pw_url,
                                  destfile = f_path,
+                                 method = "wb",
```
Comment (egeulgen) [inline R/visualization_functions.R:664]: the argument name should be `mode` not `method` (that is for "wget", "curl" etc.).

Comment (rix133) [reply R/visualization_functions.R]: of course :) I don't know how I managed to mixed this up

Comment (egeulgen) [reply R/visualization_functions.R]: easy mistake, couldn't see it for a while myself :)

CI checks on the way, will merge if successful

**Pull request merged**